result/src/main/scala-2.13+/uscala/result/ResultBuildFromImplicits.scala

```diff
@@ -0,0 +1,44 @@
+package uscala.result
+
+import uscala.result.Result.{Fail, Ok}
+
+import scala.collection.BuildFrom
+
+trait ResultBuildFromImplicits {
+  import scala.language.higherKinds
+  
+  implicit class IterableResult[E, A, M[X] <: IterableOnce[X]](xs: M[Result[E, A]]) {
+    def sequence(implicit cbf: BuildFrom[M[Result[E, A]], A, M[A]]): Result[E, M[A]] =
+      xs.iterator.foldLeft(Result.ok[E, scala.collection.mutable.Builder[A, M[A]]](cbf.newBuilder(xs))) { (fr, fa) =>
+        for {
+          r <- fr
+          a <- fa
+        } yield r += a
+      }.map(_.result())
+
+    def split(implicit bfe : BuildFrom[Seq[E], E, M[E]], bfa : BuildFrom[Seq[A], A, M[A]]): (M[E], M[A]) = {
+      val be = bfe.newBuilder(Seq.empty[E])
```

Comment:
`BuildFrom. newBuilder` does not have a version which takes no arguments anymore like the old `CanBuildFrom.apply` did - so we force the issue by creating an empty collection with the inner type of the type we want.